documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR.
+2. If the service has onboarded DPG before, please go to step 4. Otherwise, please go to step 3.
+3. Create branch with name `dpg/<prNumber>` in sdk repository, and add `autorest.md` or `README.md`. Then add a comment `/azp run` in swagger PR.
+   1. `<prNumber>` is the PR number of swagger PR.
+   2. About how to add `autorest.md` or `README.md`, please refer to [Add autorest configuration in sdk repository](./add-autorest-configuration-in-sdk-repository.md).
+4. Swagger PR CI generates SDK and ApiView automatically, and you can find them in swagger PR comments.
+5. Ask Arch Board to review the generated ApiView.
+6. If Arch Board approves the ApiView, go to step 10. Otherwise, go to step 7.
+7. Update Swagger if needed.
+8. Update `autorest.md` or `README.md` if needed. Here, if there is a branch named `dpg/<prNumber>` in sdk repository, please update it in the branch. Otherwise update it in main branch.
+9. Add a comment `/azp run` in swagger PR, and go to step 4.
+10. Get the generated SDK and do some manual changes, such as adding samples/tests to it.
+    1. Swagger PR CI creates a PR for generated SDK, and you can find it in comment `[Swagger Generation Artifacts]->[sdk-repository-name]->[Preview SDK changes]`. For example: `[Swagger Generation Artifacts]->[azure-sdk-for-js]->[Preview SDK changes]`.
+    2. About what manual changes you need to do, please refer to [Next Steps After Generating Codes](./next-steps-after-generating-codes.md).
+11. Create a PR to sdk repository and ask SDK owner to review it.
```

Comment:
Could you simplify the steps in 9,10,11 and combine them together?
i think we are mainly talk about the SDK automation instead of getstarted with SDK.

Reply:
merge 10 and 11. For 9, it ask service team to go back to step 4, and it's not fine to merge it.